agent_server.py

```
@@ -619,7 +624,15 @@ async def _do_analyze_and_plan(messages: list[dict[str, Any]], lanlan_name: Opti
                 else:
                     logger.info(f"[ComputerUse] Duplicate task detected, matched with {matched}")
             else:
-                logger.warning(f"[ComputerUse] Task requires ComputerUse but it's disabled")
+                logger.warning(f"[ComputerUse] ⚠️ Task requires ComputerUse but it's disabled")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)